src/tools/clippy/tests/ui/checked_conversions.stderr

```diff
@@ -1,52 +0,0 @@
-error: Checked cast can be simplified.
-  --> $DIR/checked_conversions.rs:13:8
-   |
-LL |     if value <= (u32::max_value() as i64) && value >= 0 {
-   |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try: `u32::try_from(value).is_ok()`
-   |
-   = note: `-D clippy::checked-conversions` implied by `-D warnings`
```

Comment:
This file shouldn't be getting removed. We still want to test these lints.

Reply:
That's a false negative clippy lints: rust-lang/rust-clippy#5675

Reply:
Ah okay. In that case could you add a comment explaining this in checked_conversions.rs, with a link to the issue? That will help the next person figure out what happened when the warnings later reappear.